### PR TITLE
Add documentation for update ecs service

### PR DESCRIPTION
**Pull request by tekerson**

### CODE REVIEW

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:10]:
```suggestion
The `Update Amazon ECS Service` step was added in Octopus **2021.3**. Presently only **Fargate** clusters are supported.
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:7]:
```suggestion
This step provides an opinionated deployment workflow that allows new releases to be deployed to an ECS Cluster that is managed externally, for example manually or via Terraform.
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:20]:
```suggestion
The following instructions can be used to configure the `Update Amazon ECS Service` step.
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:50]:
```suggestion
As the step is using a deployment target, a target role will also need to be specified.
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:59]:
```suggestion
This is the task definition that will receive a new revision upon deployment.
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:76]:
```suggestion
Specify the container name that will be used to locate the container definition within your task and select a feed and image that your task will run.
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:84]:
```suggestion
Environment files should be stored in S3, they must have a ".env" extension, and each line should contain a `VARIABLE=VALUE` variable definition.
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:95]:
```suggestion
Plain text environment variables will use the value provided, while secret variables refer to a secret value stored in AWS Secrets Manager.
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:125]:
```suggestion
The step outputs the following variables:
```

File: docs/deployments/aws/ecs-update-service/index.md
Comment (harrisonmeister) [inline docs/deployments/aws/ecs-update-service/index.md:127]: I think these variables would be better placed in a table. Something like this:

```
| Name | Description |
| -------------------- | -------|
|`TaskDefinitionRevision` | The revision number of the newly created task (if a new revision was created) |
|`TaskDefinitionFamily` | The name of a family that the new task definition is registered to (if a new revision was created) |
|`ClusterName` | The name of the cluster to which the updated service belongs |
|`Region` | The AWS region in which the operations were executed |
```
That would output like this:

| Name | Description |
| -------------------- | -------|
|`TaskDefinitionRevision` | The revision number of the newly created task (if a new revision was created) |
|`TaskDefinitionFamily` | The name of a family that the new task definition is registered to (if a new revision was created) |
|`ClusterName` | The name of the cluster to which the updated service belongs |
|`Region` | The AWS region in which the operations were executed |

**Review by harrisonmeister — COMMENTED** [review]
Made some minor line edit suggestions

File: docs/deployments/aws/ecs-update-service/index.md
Comment (tekerson) [reply docs/deployments/aws/ecs-update-service/index.md:127]: Yep, I did consider making it a table but I didn't know how people felt about them. The format I used is supposed to be a definition list but it's an extended markdown syntax and I wasn't sure if our parser/renderer supports it.

I'll change it for a table.

File: docs/deployments/aws/ecs-update-service/index.md
Comment (tekerson) [inline docs/deployments/aws/ecs-update-service/index.md:140]:
```suggestion
| Name | Description |
| -------------------- | -------|
|`TaskDefinitionRevision` | The revision number of the newly created task (if a new revision was created) |
|`TaskDefinitionFamily` | The name of a family that the new task definition is registered to (if a new revision was created) |
|`ClusterName` | The name of the cluster to which the updated service belongs |
|`Region` | The AWS region in which the operations were executed |
```

**Review by harrisonmeister — APPROVED** [review]
LGTM

**Comment by harrisonmeister** [comment]
@tekerson - I'm happy to merge, or if you want to that's cool. I wasn't sure if you were making any additional changes :+1:

**Comment by tekerson** [comment]
> @tekerson - I'm happy to merge, or if you want to that's cool. I wasn't sure if you were making any additional changes 👍

@harrisonmeister - No, not waiting on any further changes from my side but the feature isn't finalised/released yet so I was holding off on merging the docs until it is.

**Comment by egorpavlikhin** [comment]
@harrisonmeister the step is now in master, could you please merge the PR if you are happy with the recent changes?

**Pull request merged**
